vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -146,6 +221,24 @@ void ScrollViewShadowNode::updateProperties(winrt::Microsoft::ReactNative::JSVal
         m_isScrollingEnabled = scrollEnabled;
         SetScrollMode(scrollViewer);
       }
+    } else if (propertyName == "isFlatList") {
+      const auto [valid, isFlatList] = getPropertyAndValidity(propertyValue, true);
+      if (valid && isFlatList) {
+        // We use the XAMl ScrollViewer for both the RN ScrollView and FlatList - need arrow navigation of FlatList
+        // entries.
+        m_PreviewKeyDownRevoker = scrollViewer.PreviewKeyDown(
+            winrt::auto_revoke, [=](const winrt::IInspectable &sender, const xaml::Input::KeyRoutedEventArgs &e) {
```

Comment:
Implicit capture by value (the `[=]`) should generally be avoided because it makes it too easy to introduce subtle memory safety issues or ref cycles. Reminded of this after a PR came in today for this happening (https://github.com/microsoft/react-native-windows/pull/6592) 😀.

So, for this specific case, we're relying on the raw `this` ptr inside the lambda, and the equivalent would be to replace `[=]` with `[this]`. Capturing a raw pointer in an async lambda generally best to avoid compared to something with weak owership, but is safe here because the revoker ensures the callback will be unregistered before the object is destroyed.